datagen/data/departments.json

```diff
@@ -37443,7 +37443,7 @@
         "description": "The roles of the sales executive as a planner, manager of resources and marketing functions and recruiter, trainer, motivator and leader of field sales personnel. Techniques and sequence of problem analysis for selling and sales management decisions and to the practical framework in which these decisions take place. Teaching vehicles feature strong classroom interactions, cases, journal articles, research findings, guest sales managers, debates, and modern company practices.",
         "sections": [
           {
-            "sec_code": "1",
+            "sec_code": "G1",
```

Comment:
This section code is also not correct: [BMGT455](https://app.testudo.umd.edu/soc/search?courseId=bmgt455&sectionId=&termId=202401&_openSectionsOnly=on&creditCompare=%3E%3D&credits=0.0&courseLevelFilter=ALL&instructor=&_facetoface=on&_blended=on&_online=on&courseStartCompare=&courseStartHour=&courseStartMin=&courseStartAM=&courseEndHour=&courseEndMin=&courseEndAM=&teachingCenter=ALL&_classDay1=on&_classDay2=on&_classDay3=on&_classDay4=on&_classDay5=on); it should be ESG1 instead of G1.